pkg/flowkit/util/mixpanelEvent.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package util
+
+import (
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+const (
+	MIXPANEL_EVENT_PROJECT_TOKEN = "token"
+	MIXPANEL_EVENT_CALLER        = "caller"
+
+	FLOW_CLI               = "flow-cli"
+	MIXPANEL_PROJECT_TOKEN = "da53727e0435e12820c831098691f8e5"
+)
+
+type event struct {
+	Name       string                 `json:"event"`
+	Properties map[string]interface{} `json:"properties"`
+}
+
+func newEvent(command *cobra.Command) *event {
+	return &event{
+		processCommandName(command),
+		make(map[string]interface{}),
+	}
+}
+
+//transform command path name e.g. flow accounts get -> accounts-get
+func processCommandName(command *cobra.Command) string {
```

Comment:
I feel it would be nicer if we get the whole command as it is eg: flow accounts get than adding the dashes, because some of the commands have dashes in names like: flow transactions send-signed which then becomes transactions-send-signed and it's not clear what it is. The only thing worth modifying is stripping the first flow, but even that I don't mind. Also why are you checking if first command is flow is that ever not true?

Reply:
Also if you plan to keep this method in order to strip `flow` you should make it an event method. And just make `name` unexported and make this a getter, so this would be calculated when you call `event.Name()`

Reply:
addressed, we will just be used the whole command path, i.e. flow accounts get